### PR TITLE
Added Emoji Component

### PR DESCRIPTION
**Pull request by fleonard**
## Screenshot

Screenshot 2021-01-11 at 13 27 55

## What does this do?

Added new component to properly render an emoji into a span with option to change size and add an a11y friendly label

### CODE REVIEW

File: src/Emoji/Emoji.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+type EmojiProps = {
+  /** className attribute to apply classes from props */
+  className?: string
+  /** Emoji symbol */
+  symbol: string
+  /** set size of the Emoji  */
+  size?: number
+  /** aria-label to identify Emoji */
+  label?: string
```
Comment (alexwllms) [inline src/Emoji/Emoji.tsx:12]: Do you think this should be enforced rather than optional to avoid a bunch of "emoji" strings?

File: src/Emoji/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# < Emoji />
+
+## Props:
+
+```ts
+symbol:   string;
```
Comment (alexwllms) [inline src/Emoji/README.md:6]: Should we clarify that you have to add (or copy and paste) the emoji in directly as a string rather than it's text description?

File: src/Icon/README.md [hunk 3]
```diff
@@ -65,4 +65,4 @@ const App = () => (
 * clock
 * up-down
 * upload-email
-* warning
\ No newline at end of file
+* warning
```
Comment (alexwllms) [inline src/Icon/README.md:68]: Just checking that you meant to do the changes in this file as they look unrelated? If so ignore me.

**Review by alexwllms — APPROVED** [review]
A few comments but well done! You are doing god's work @fleonard

File: src/Icon/README.md (hunk 3 above)
Comment (fleonard) [reply src/Icon/README.md:68]: yeah I've just updated the comment since it was a copy paste from the Box component so I've updated it

File: src/Emoji/README.md (hunk 2 above)
Comment (fleonard) [reply src/Emoji/README.md:6]: Good point
